### ISSUE TITLE
Questions about results and Deduplication processing

### ISSUE THREAD

**Issue by Li-Qingyun** [NJU-LHRS/ScoreRS#2]
Hello, I find your work and technical readmap truly impressive after reading your paper and blog. I have two questions:

- It seems that the last result in Table 3 and the last result in Table 4 represent the same setting (i.e., 30% pre-training and 60% fine-tuning)? Is this right?
![Image](https://github.com/user-attachments/assets/bf26e61b-06fe-4e78-bd5e-3734de963d86)
- I actually did not understand the two `Deduplication` processing clearly, it seems that it decrease the data quantity but limit the data diversity for training the reward model. Does it resolve some of specific problems hidden in RS instruction data?

**Comment by pUmpKin-Co** [comment]
Hi~ Thank you for your interest!

1. Yes.
2. To be honest, the most important reason behind the dudplication process is reducing the cost for labeling and scoring. But in other hand, since we know that most RS VL data is constructed from some publick benchmark (e.g., classification, etc.), most prompts (aka, quries) and images (same categories in classification) may occupy most amount of data and results our RM bias to this distribution. But I haven't explore and ablate this and hope further study could analysis this!

**Comment by Li-Qingyun** [comment]
Hi, thanks for your quick reply.  I am very interested in your research, so please forgive me for asking questions in a manner as meticulous as a reviewer.
 
1. So, they are just two different runnings of the same experiments setting. actually they should be the same?
2. Okkkk, so what does the cost look like? 
3. (new): May there be a conclusion?: the proposed automatic preference data collecting method (with human-aligned GPT-4o) is almost aligned with the preference of human expert (92% agreement?) on image captioning (or just remote sensing image captioning)?
4. (new): For the caption task, the used GPT-4o is aligned with human preference. But it seems that the Qwen72B instruct models have not been aligned with human preference. Besides, the similar assessment is not conducted for instruction subset.
5. (new): As a newcomer to RL for mllm, I initially thought that the proposed synthetic preference data would be used in methods like DPO/MPO. I'm curious about how the quality of the preference data we proposed holds up in more beginner-friendly strategies like DPO.

（Self-reflection: Indeed, the paper (particularly in its title) has clearly elucidated that the crux of this work lies in the conclusions drawn from data filtering. However, the hot RL-related topics designed within the paper are exceptionally captivating. It seems to me that this is a sentiment that subsequent reviewers will likely share.）

**Comment by pUmpKin-Co** [comment]
Hi! Thank you for your thoughtful questions.

1. After data ablation, we sclale up the LoRA rank for matching the results of different VLM counterparts.
2. For GPT4o inference, we usually consume average 1600+ token for input, and the output token is usually 500, so it takes 💲 (0.004 + 0.005) for scoring a single sample (where we also use the batch inference API for reducing the cost). For developing Qwen2VL-70B, we consider the usage of TogetherAI or Ali Cloud API, but the generation speed is far from statisfaction. Therefore, we customize the serving with VLLM on 2 A100 node. The juge process with Qwen2VL-70B takes roughly 2 weeks to finish (after several resuming with unknown error).
3. We just cross validate with human preference on samples from image captions and RS specfic questions. Other juding process we give the judger the reference answer from its source dataset, therefore we do not take effort to validate it.
4. I should see thatm from our experience, Qwen2VL-70B (and of course, Qwen2.5VL) or InternVL2.5-72B perform onpar or even outperform the GPT4o. As for the human preference, I am not sure whether GPT4o conduct somewhat preference alignment sicne they have not release their technical details. Although the report from Qwen2VL dosen't show they perform somewhat PO, but from our experience, it indeed is a realiable choice given our limited resources.
5. Indeed, we could perform DPO. 
    And actually, we internally have a DPO tuned model from our LHRS-Bot-Nova, but I should admit the observation in my blog still exists. 
    Moreover, I do not think use our preference data for performing DPO will be a good choice. Since the perferecne data is collected from base Qwen2VL, LHRS-Bot-Nova or InternVL. But our Qwen2VL-RS have been fintuned and the preference data has become "off-policy" data. The clever choice is use the Qwen2VL-RS to perform somewhat rejection sampling to collect new preference data for DPO. 
   The process indeed deserve to explore too, but just as my blog said. I just want to make our model to perfer to answer "good" answer and our model indeed has the ability to output "good" answer. Why not directly use its own output and reward itself online instead of collecting offline preference data?


Yes. The RL part of our work may more eye cathing since the current RL storm. But remember, the prerequist of RL is a strong base model. And one of the prerequist for a strong base model is the data.

**Comment by Li-Qingyun** [comment]
OKKKK, thank you for the your sincere response.
